app/controllers/admin/npq/applications/eligibility_status_controller.rb

```diff
@@ -6,43 +6,47 @@ module Applications
       class EligibilityStatusController < Admin::BaseController
         StatusOption = Struct.new(:value, :label)
 
-        skip_after_action :verify_authorized
         skip_after_action :verify_policy_scoped
 
+        before_action :assign_npq_application
+
         def edit
           @status_options = [
-            StatusOption.new(:marked_ineligible_by_policy, "Not eligible for funding"),
-            StatusOption.new(:awaiting_more_information, "Awaiting more information"),
-            StatusOption.new(:re_register, "Needs to re-register"),
+            StatusOption.new(:marked_ineligible_by_policy, t(".marked_ineligible_by_policy")),
+            StatusOption.new(:awaiting_more_information, t(".awaiting_more_information")),
+            StatusOption.new(:re_register, t(".re_register")),
           ]
-
-          @npq_application = NPQApplication.find(params[:id])
         end
 
         def update
-          npq_application = NPQApplication.find(params[:id])
-          npq_application.assign_attributes(eligiblity_status_params)
+          @npq_application = NPQApplication.find(params[:id])
```

Comment:
If we've set up assign_npq_application then we can drop this duplicate loading right?

Reply:
mistake yes will remove